packages/shopify-codemod/test/fixtures/remove-empty-catch-blocks/basic.output.js

```diff
@@ -0,0 +1,15 @@
+try {
+  foo();
+} finally {
+  bar();
+}
+
+try {
+  foo();
+} finally {
+  bar();
+}
+
+try {
+  foo();
+} finally {}
```

Comment:
We are missing a `try` with a `catch` but without a `finally` — The catch can't be removed in that case.